docs/source/index.rst

```diff
@@ -1,18 +1,19 @@
-.. Hive Metastore Client documentation master file.
+Hive Metastore Client
+=====================
+Made with |:heart:| by the **Data Engineering** team from `QuintoAndar <https://github.com/quintoandar/>`_.
 
-Welcome to Hive Metastore Client's documentation!
-=============================================================================================
+A client for connecting and running DMLs on Hive Metastore using Thrift protocol.
 
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
+An example of how to use the library for running DML commands in hive metastore:
```

Comment:
Where is the example? :thinking:

Reply:
rs. I had to remove it because it uses ` ``` ` and it's rest file. I'm thinking about using a MD instead. If it works I'll return the example. Else I remove this line. Tks!

Reply:
you could move it to getstarted.md instead :thinking:

Reply:
resolved it =]